qdrant-landing/content/documentation/fastembed/fastembed-colbert.md

```diff
@@ -5,33 +5,58 @@ weight: 6
 
 # How to Generate ColBERT Multivectors with FastEmbed
 
-With FastEmbed, you can use ColBERT to generate multivector embeddings. ColBERT is a powerful retrieval model that combines the strength of BERT embeddings with efficient late interaction techniques. FastEmbed will provide you with an optimized pipeline to utilize these embeddings in your search tasks.
+Qdrant supports [multivector representations](https://qdrant.tech/documentation/concepts/vectors/#multivectors). 
+With FastEmbed, you can use ColBERT to generate multivector embeddings; FastEmbed will provide an optimized pipeline to utilize these embeddings in your search tasks.
```

Comment:
It feels like these 3 statements lack a connection link between them

Reply:
I might have written this back in the day, but it's too dry. A more detailed explanation for regular people is needed.